### PR TITLE
fix typo: Samir -> Shamir

### PR DESCRIPTION
**Pull request by sellibitze**

### CODE REVIEW

File: content/2015-02-02-this-week-in-rust.md [hunk 1]
```diff
@@ -233,4 +233,4 @@ Now you can follow breaking changes *[as they happen][BitRust]*!
 
 [n]: http://www.meetup.com/N-Languages-in-N-Months-NYC/events/218757056/
 [syd]: http://www.meetup.com/Rust-Sydney/events/220100853/
-[seattle]: https://www.eventbrite.com/e/mozilla-rust-seattle-meetup-tickets-12222326307?aff=erelexporg
\ No newline at end of file
+[seattle]: https://www.eventbrite.com/e/mozilla-rust-seattle-meetup-tickets-12222326307?aff=erelexporg
```
Comment (Gankro) [inline content/2015-02-02-this-week-in-rust.md]: What happened here? line-ending whitespace or something?

Comment (Gankro) [inline content/2015-02-02-this-week-in-rust.md]: Eh it's in another PR, so I guess that's it.

Comment (Gankro) [inline content/2015-02-02-this-week-in-rust.md]: Oh weird, in the PR view it shows something different!

**Comment by Gankro** [comment]
Thanks!

File: content/2015-02-02-this-week-in-rust.md (hunk 1 above)
Comment (sellibitze) [inline content/2015-02-02-this-week-in-rust.md]: Yeah, my editor added a line feed for the last line. I noticed this a bit too late.

**Pull request merged**